File: java-play-framework/app/apimodels/UserNotAuthorizedError.java
package apimodels;

import apimodels.CodedError;
import com.fasterxml.jackson.annotation.*;
import java.util.Set;
import javax.validation.*;
import java.util.Objects;
import javax.validation.constraints.*;
/**
 * UserNotAuthorizedError
 */
@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaPlayFrameworkCodegen", date = "2021-08-08T21:11:44.583112Z[Etc/UTC]")
@SuppressWarnings({"UnusedReturnValue", "WeakerAccess"})
public class UserNotAuthorizedError   {
  @JsonProperty("result")
  @NotNull

  private Object result = null;

  @JsonProperty("msg")
  @NotNull

  private Object msg = null;

  @JsonProperty("code")
  
  private Object code = null;

  public UserNotAuthorizedError result(Object result) {
    this.result = result;
    return this;
  }

   /**
   * Get result
   * @return result
  **/
  public Object getResult() {
    return result;
  }

  public void setResult(Object result) {
    this.result = result;
  }

  public UserNotAuthorizedError msg(Object msg) {
    this.msg = msg;
    return this;
  }

   /**
   * Get msg
   * @return msg
  **/
  public Object getMsg() {
    return msg;
  }

  public void setMsg(Object msg) {
    this.msg = msg;
  }

  public UserNotAuthorizedError code(Object code) {
    this.code = code;
    return this;
  }

   /**
   * Get code
   * @return code
  **/
  public Object getCode() {
    return code;
  }

  public void setCode(Object code) {
    this.code = code;
  }


  @Override
  public boolean equals(Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    UserNotAuthorizedError userNotAuthorizedError = (UserNotAuthorizedError) o;
    return Objects.equals(result, userNotAuthorizedError.result) &&
        Objects.equals(msg, userNotAuthorizedError.msg) &&
        Objects.equals(code, userNotAuthorizedError.code);
  }

  @Override
  public int hashCode() {
    return Objects.hash(result, msg, code);
  }

  @SuppressWarnings("StringBufferReplaceableByString")
  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class UserNotAuthorizedError {\n");
    
    sb.append("    result: ").append(toIndentedString(result)).append("\n");
    sb.append("    msg: ").append(toIndentedString(msg)).append("\n");
    sb.append("    code: ").append(toIndentedString(code)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: cpp-tiny/lib/Models/UserNotAuthorizedError.cpp


#include "UserNotAuthorizedError.h"

using namespace Tiny;

UserNotAuthorizedError::UserNotAuthorizedError()
{
	result = null;
	msg = null;
	code = null;
}

UserNotAuthorizedError::UserNotAuthorizedError(std::string jsonString)
{
	this->fromJson(jsonString);
}

UserNotAuthorizedError::~UserNotAuthorizedError()
{

}

void
UserNotAuthorizedError::fromJson(std::string jsonObj)
{   
    bourne::json object = bourne::json::parse(jsonObj);

    const char *resultKey = "result";
    
    if(object.has_key(resultKey)) 
    {
        bourne::json value = object[resultKey];


        

        AnyType* obj = &result;
		obj->fromJson(value.dump());

    }

    const char *msgKey = "msg";
    
    if(object.has_key(msgKey)) 
    {
        bourne::json value = object[msgKey];


        

        AnyType* obj = &msg;
		obj->fromJson(value.dump());

    }

    const char *codeKey = "code";
    
    if(object.has_key(codeKey)) 
    {
        bourne::json value = object[codeKey];


        

        AnyType* obj = &code;
		obj->fromJson(value.dump());

    }


}

bourne::json
UserNotAuthorizedError::toJson()
{
    bourne::json object = bourne::json::object();

    




	object["result"] = getResult().toJson();

    




	object["msg"] = getMsg().toJson();

    




	object["code"] = getCode().toJson();


    return object;

}

AnyType
UserNotAuthorizedError::getResult()
{
	return result;
}

void
UserNotAuthorizedError::setResult(AnyType  result)
{
	this->result = result;
}

AnyType
UserNotAuthorizedError::getMsg()
{
	return msg;
}

void
UserNotAuthorizedError::setMsg(AnyType  msg)
{
	this->msg = msg;
}

AnyType
UserNotAuthorizedError::getCode()
{
	return code;
}

void
UserNotAuthorizedError::setCode(AnyType  code)
{
	this->code = code;
}



